lib/config-loader.js

```diff
@@ -3,6 +3,7 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+var cloneDeep = require('lodash').cloneDeep;
```

Comment:
another note on lodash, we had [previous goals to shrink bundle size](https://github.com/strongloop/loopback/issues/989), would this be going against that, 
but CI being more critical, another option would be: we are using [`traverse.clone()`](https://www.npmjs.com/package/traverse) in [datasource-juggler](https://github.com/strongloop/loopback-datasource-juggler/blob/6bc44418509a7269cdd2e862d806a091ec173003/lib/datasource.js#L427)